src/components/MobileWalletDialog/routes/CreateWallet/CreateBackupConfirm.tsx

```diff
@@ -21,103 +21,86 @@ import {
 } from '@/components/Modal/components/DialogHeader'
 import { SlideTransition } from '@/components/SlideTransition'
 
-const makeOrdinalSuffix = (n: number) => {
-  return ['st', 'nd', 'rd'][((((n + 90) % 100) - 10) % 10) - 1] || 'th'
-}
-
 const TEST_COUNT_REQUIRED = 3
 
 export const CreateBackupConfirm = () => {
   const navigate = useNavigate()
   const location = useLocation()
   const translate = useTranslate()
-  const borderColor = useColorModeValue('gray.100', 'gray.700')
-  const [selectedWordIndex, setSelectedWordIndex] = useState<number | null>(null)
-  const [testWords, setTestWords] = useState<string[]>([])
-
-  const backgroundDottedSx = useMemo(
-    () => ({
-      content: '""',
-      position: 'absolute',
-      top: 0,
-      left: 0,
-      right: 0,
-      bottom: 0,
-      borderWidth: 1,
-      borderStyle: 'dashed',
-      borderColor,
-      borderRadius: 'xl',
-      mask: 'linear-gradient(to bottom, black 20%, transparent 100%)',
-      WebkitMask: 'linear-gradient(to bottom, black 20%, transparent 100%)',
-    }),
-    [borderColor],
+  const [targetIndices, setTargetIndices] = useState<number[]>([])
+  const [selections, setSelections] = useState<(string | null)[]>(
```

Comment:
q: do we really need to fill this with empty elements? Most bits here check for `selections.length >=3` but this will always evaluate to true, indicating that we may not want to prefill?

Reply:
Good catch, we obviously don't need, used an empty array in [5346ecc](https://github.com/shapeshift/web/pull/9756/commits/5346ecc14a9fa14f7a774fde5061e58b867b96a4)